_data/WahajJaved/WahajJaved.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Wahaj Javed
+institution: National University of Computer and Emerging Sciences 
+profile_pic: wahajaved.png
```

Comment:
Incorrect file name, please do one of the following:
1. Change `wahajaved.png` to `WahajJaved.jpg` in the markdown file.
2. Rename `WahajJaved.jpg` to `wahajaved.png`.
I'd suggest you make the change in the markdown file!